templates/repo/settings/options.tmpl

```diff
@@ -19,7 +19,11 @@
 					<div class="inline field">
 						<label>{{.i18n.Tr "repo.visibility"}}</label>
 						<div class="ui checkbox">
+							{{if .IsAdmin}}
+							<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}>
+							{{else}}
 							<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}{{if and $.ForcePrivate .Repository.IsPrivate}} readonly{{end}}>
```

Comment:
To follow the same check/logic as in `routers/repo/setting.go` you just need to add (not .IsAdmin) to the and list.
```suggestion
							<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}{{if and $.ForcePrivate .Repository.IsPrivate (not .IsAdmin)}} readonly{{end}}>
```

Reply:
I have just seen the comment of @zeripath and I will LGTM as it is not blocking.